test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -170,12 +171,23 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 			case 'Markdown':
 				prefix = 'jetpack-';
 				break;
+			case 'Buttons':
+			case 'Click to Tweet':
+			case 'Hero':
+			case 'Logos':
+			case 'Pricing Table':
+				prefix = 'coblocks-';
+				break;
+			case 'Dynamic HR':
+				prefix = 'coblocks-';
+				blockClass = 'dynamic-separator';
```

Comment:
This was necessary because the block's selector is `button.editor-block-list-item-coblocks-dynamic-separator`, while its name is Dynamic HR 🙃